react/react1/week3/vite-project/src/todo-list.jsx

```diff
@@ -0,0 +1,42 @@
+import ListItem from "./todo-list-item";
+import "./index.css";
+import { useState , useEffect} from "react";
+
+const List = ({ todosList }) => {
+  debugger;
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const [todos, setTodos] = useState([]);
```

Comment:
Again, we can make this component stateless by passing functions as props from the parent component